src/bash/unittest/Makefile

```diff
@@ -0,0 +1,14 @@
+#disable some warning because UT need test functions not in header file.
+CFLAGS = -Wno-parentheses -Wno-format-security -Wno-implicit-function-declaration -c
+IFLAGS = -I.. -I../include -I../lib
+MFLAG = -DDEBUG -DBASH_PLUGIN_UT
+
+all:
```

Comment:
>all [](http://example.com/codeflow?start=0&length=3)

How should we make sure it is tested in every build?

Reply:
Will check how other gtest work.